lib/saluki-components/src/sources/checks/execution_context.rs

```diff
@@ -0,0 +1,73 @@
+use std::collections::HashMap;
+
+use saluki_config::GenericConfiguration;
+use saluki_env::{EnvironmentProvider, HostProvider};
+use saluki_metadata;
+use tracing::warn;
+
+// Cache execution information from datadog agent for Python checks
```

Comment:
```suggestion
/// Global/shared configuration for checks.
```

In Rust, "doc" comments are indicated by three forward slashes (instead of the traditional two), which means they get consumed and used for things like IDE hover hints, generated documentation (`docs.rs`, etc), and so on... so any time we're adding documentation on public types/methods/fields, we want to use the triple forward slash format.

I've also just provided a slightly more generic description here, as we try to avoid specifically always referring back to the Datadog Agent in terms of what something is related to.

Reply:
Thanks for the explanation and improvement!